Allow simple SQS queue name (#628)
kedacore/keda-docs: content/docs/2.6/scalers/aws-sqs.md

**Parameter list:**

> - `queueURL` - Full URL for the SQS Queue. The simple name of the queue can be used in case there's no ambiguity.

Reviewer: Hm, shouldn't we go with this instead?
```suggestion
- `queueURL` - Full URL for the SQS Queue. (Optional, when `queueName` is specified)
- `queueName` - Name of the queue.  (Optional, when `queueURL` is specified)
```

Author: I'm not sure. At API level you can use a simple name in the queueURL field, so I don't know if a new field is needed.

Reviewer: From a KEDA user-perspective they don't know that and providing `orders-queue` in `queueURL` would be confusing, no?

Reviewer 2: There is not any extra parameter here @tomkerkhove. The behavior of current `queueURL` parameter has changed because it's not needed "the full name". Am I right @nicolaferraro ?

Reviewer 2: Maybe we could transform `queueURL` into `queueName` and explain that it could be full qualified name or simple name. WDYT?

Reviewer: That would be a breaking change though. I'd just provide seperate fields and internally you can just use the same.

From a user-perspective, there's no added value to try to cram them both in one field - No?

Reviewer 3: I think it is OK, to keep it as it is. There's no need to actually create a new property imho, though the naming is not ideal, I know :/ But it is better than actually introducing a breaking change.

Reviewer: It wouldn't really be breaking if we leave `queueURL` what it is and add `queueName` next to it; but let's go with the current approach as I'm the only one preferring this.
